graphkb/constants.py

```diff
@@ -54,3 +88,33 @@
     'Val': 'V',
     'Ter': '*',
 }
+
+
+class IterableNamespace(argparse.Namespace):
```

Comment:
This is minor but it seems like there's few cases where these instances are used as anything other than a normal dictionary - the tests and the gsc report occasionally do something like `INPUT_COPY_CATEGORIES.AMP` but that's about it.

If we ever want to reduce complexity a bit we could convert `INPUT_COPY_CATEGORIES` and `INPUT_EXPRESSION_CATEGORIES` to regular dictionaries.

Correct me if I've overlooked something though.

Reply:
That might be correct.

I was somewhat unsure and left them here for now.